src/js/tabs/osd.js

```diff
@@ -1626,6 +1631,7 @@ OSD.chooseFields = function () {
             F.RSSI,
             F.LINK_QUALITY,
             F.RSSI_DBM,
+            F.OVER_CAP,
```

Comment:
If this is for Betaflight 4.2, it needs to be done only for MSP version 1.43.

Reply:
Ah I wasn't sure about that, it didn't seem visible on a quad that didn't have support for the warning.